bodo/tests/test_iceberg/test_iceberg_theta.py

```diff
@@ -226,12 +227,11 @@ def check_no_statistics_file(warehouse_loc, db_schema, table_name):
     )
 
 
-@bodo.jit
+@numba.njit
 def get_statistics_ndvs(puffin_file_name, iceberg_schema):
     return read_puffin_file_ndvs(puffin_file_name, iceberg_schema)
 
 
-@run_rank0
```

Comment:
Serialization of schema in MPI broadcast returns a different data type only on Windows and with bash for some reason. Doesn't seem worth investigating right now.